app/components/header.jsx

```diff
@@ -8,11 +8,11 @@ import moon from "../images/moon.png"
 
 export default function Header({ isDarkMode, setIsDarkMode }) {
     return (
-        <div className="flex items-center justify-between">
+        <div className="flex items-center justify-between w-screen absolute">
             <div className="font-bold text-lg mt-4">
                 <Link className="text-2xl ml-6 mr-10 text-white hover:underline" href="/">Shop</Link>
                 <Link className="text-2xl mr-10 text-white hover:underline" href="https://acm.uic.edu/about">About</Link>
-                <Link className="text-2xl text-white hover:underline w" href='https://acm.uic.edu/about'>Contact</Link>
+                <Link className="text-2xl text-white hover:underline w" href='/Contributors'>Contact</Link>
```

Comment:
Keep the contact link to https://acm.uic.edu/contact. The link the website contributors will be added to the footer.

Reply:
i made a commit to the footer branch which changed it back to acm.uic.edu/contact